inference/enterprise/parallel/Dockerfile.onnx.cpu.parallel

```diff
@@ -0,0 +1,65 @@
+FROM python:3.9
```

Comment:
Do you see gains using the parallel architecture with CPU only? And if so, is that due to converting all the calls to async?

Reply:
I haven't found time to test cpu only yet but I would imagine we'd see some gains!